src/node/node-utils.ts

```diff
@@ -80,34 +93,116 @@ export function updateMoveToml(
     packageName: string,
     packageId: string,
     moveDir: string = `${__dirname}/../../move`,
-    prepToml: undefined | ((moveJson: Record<string, TomlPrimitive>) => Record<string, TomlPrimitive>) = undefined,
+    prepToml: undefined | ((tomlJson: Record<string, TomlValue>) => Record<string, TomlValue>) = undefined,
+    // Version should be the 0 indexed variant (as per axelar-contract-deployments chain config)
+    version?: undefined | number,
+    network?: undefined | string,
+    originalPackageId?: undefined | string,
 ) {
+    if (typeof version !== 'number') {
+        version = 0;
+    }
+
+    if (typeof network !== 'string') {
+        network = 'testnet';
+    }
```

Comment:
why allow a default? require it?

Reply:
if we require this parameter it's no longer backwards compatible